documentation/puppet_server_metrics_performance.markdown

```diff
@@ -88,3 +92,5 @@ Puppet Server also requests facts as HTTP requests while handling a node request
 A memory leak or increased memory pressure can stress Puppet Server's available resources. In this case, the Java VM will spend more time doing garbage collection, causing the GC time and GC CPU % metrics to increase. These metrics are available from the [status API][] endpoint, as well as in the mbeans metrics available from both the [`/metrics/v1/mbeans`](./metrics-api/v1/metrics_api.markdown) or [`/metrics/v2/`](./metrics-api/v2/metrics_api.markdown) endpoints.
 
 If you can't identify the source of a memory leak, setting the `max-requests-per-instance` setting in [`puppetserver.conf`][puppetserver.conf] to something other than the default of 0 limits the number of requests a JRuby handles during its lifetime and enables automatic JRuby flushing. Enabling this setting reduces overall performance, but if you enable it and no longer see signs of persistent memory leaks, check your module code for inefficiencies or memory-consuming bugs.
+
+> **Note:** In multithreaded mode, the `max-requests-per-instance` setting refers to the sum total number of requests processed by the single JRuby instance, across all of its threads. While that single JRuby is being flushed, all requests will suspend until the instance becomes available again.
```

Comment:
```suggestion
> **Note:** In `multithreaded` mode, the `max-requests-per-instance` setting refers to the sum total number of requests processed by the single JRuby instance, across all of its threads. While that single JRuby is being flushed, all requests will suspend until the instance becomes available again.
```
(nit)

Reply:
This is inconsistent right now, but I think we should only use the backticks when referring to the setting. When just describing the mode, I think they should be left off. I can fix the other places if that makes sense to folks?

Reply:
That works for me, as long as we're consistent. 😊